src/seed/seed.ts

```diff
@@ -0,0 +1,41 @@
+import { faker } from '@faker-js/faker';
+import prisma from '../prisma';
+import bcrypt from 'bcrypt';
+
+async function seed() {
+  try {
+    await prisma.user.deleteMany();
+
+    const User = [];
```

Comment:
```suggestion
    const users = [];
```

O `camelCase` por conta da convenção de nomenclatura de variáveis locais e o plural porque é um array.